### PR TITLE
First Pull Request

### PR DESCRIPTION
**Pull request by briafassler9**
I had a little trouble getting the pull request in which is why I have multiple commits. Also, I'm not sure about how much code to commit. This is my current progress on the Portfolio webpage. I have one out of a planned 3 pages mostly complete.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,12 +7,54 @@
...
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+    <header>
+        <div class="homeheader">
+            <h1>Bria Fassler's Portfolio</h1>
+        </div>
```
Comment (kennzoid) [inline portfolio/src/main/webapp/index.html:13]: nit: Keep indentation consistent with the rest of the file (two spaces).

*nit is short for nitpicking, which just means this is a super minor fix

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -6,4 +6,49 @@
...
 #greeting-container {
   margin-top: 20px;
+}     
+
+.homeheader {
+    text-align: center;
+    font-size: 17px;
+    background-color: #99ffeb;
+    padding: 35px;
+    color: black;
```
Comment (kennzoid) [inline portfolio/src/main/webapp/style.css:16]: nit: Keep indentation consistent.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,2 @@
-# Google Student Training in Engineering Program
+#This repo contains Bria Fassler's portfolio and STEP projects.
```
Comment (wangtong-g) [inline README.md]: Super nitpicking: Adds a space after "#".

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -7,12 +7,54 @@
...
+                <a href="https://www.linkedin.com/in/briafassler">Linkedin</a><br><br>
+                <a href="https://github.com/briafassler9/step.git">Github</a>
+                <p>Email: brifass11@aol.com</p>
+                <p>Phone Number: (443) 680-3550</p>
+        </div>
+    </aside>
+    <section>
+        <h1>About Me</h1>
+            <p>
+            Hi! Here are the basics about me. My name is Bria Fassler and I'm from Baltimore, Maryland. I'm currently 18 years old but on June 22nd, I'll be 19 so I'd say it's close enough.
```
Comment (kennzoid) [inline portfolio/src/main/webapp/index.html:34]: nit: Remove whitespace at the end of this line.

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -7,12 +7,54 @@
...
+                <a href="https://github.com/briafassler9/step.git">Github</a>
+                <p>Email: brifass11@aol.com</p>
+                <p>Phone Number: (443) 680-3550</p>
+        </div>
+    </aside>
+    <section>
+        <h1>About Me</h1>
+            <p>
+            Hi! Here are the basics about me. My name is Bria Fassler and I'm from Baltimore, Maryland. I'm currently 18 years old but on June 22nd, I'll be 19 so I'd say it's close enough. 
+            I'm pretty short, 5'1" to be exact, but I'm still hoping for a last minute growth spurt. I live with my mom and dad, but have an older sister who moved out a couple of years ago.
```
Comment (kennzoid) [inline portfolio/src/main/webapp/index.html:35]: nit: Remove whitespace at the end of this line.

I'm still waiting for a growth spurt too ):

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -6,4 +6,49 @@
 
 #greeting-container {
   margin-top: 20px;
+}
```
Comment (kennzoid) [inline portfolio/src/main/webapp/style.css:9]: nit: Remove trailing whitespace

**Pull request merged**